UpdatedImageProcessing/Classifiers/settings.py

```diff
@@ -0,0 +1,20 @@
+class ImgProcSettings(object):
+	"""
+	Settings used across all classifiers.
+	"""
+
+	LOGGING_ON = False
+
+	COLORS = {
+		"white": [255, 255, 255, 255],
+		"black": [0, 0, 0, 255],
+		"gray": [128, 128, 128, 255],
+		"red": [255, 0, 0, 255],
+		"blue": [0, 0, 255, 255],
+		"green": [0, 255, 0, 255],
+		"yellow": [255, 255, 0, 255],
+		"purple": [128, 0, 128, 255],
+		"brown": [165, 42, 42, 255],
+		"orange": [255, 165, 0, 255]
```

Comment:
If you have colors here, you should not define them in ```classify_color.py```

Reply:
Loaded colors in `classify_color.py` from `settings.py` in https://github.com/FlintHill/SUAS-Competition/pull/97/commits/5b99f5dba88c8915e7270ad090163a92629686c1